buildtest/schemas/definitions.schema.json

```diff
@@ -573,6 +572,11 @@
         "state": {
           "$ref": "#/definitions/state",
           "description": "explicitly mark state of test regardless of status calculation"
+        },
+        "mode": {
+          "type": "string",
+          "description": "Determine how the status check is resolved, for instance it can be logical AND or OR",
+          "enum": ["or", "and"]
```

Comment:
@wyphan so i would like to get your input we can basically have any keyword i proposed

we have `mode: [or|and]` but if you want to have other values we could do `OR` `AND` or `|` and `&` though i dont want to have too many values it will make it quite difficult to keep track of it. It could also be `any` `and` which map to the python built-in.

Do you have any preference or you are okay with the proposed solution.

Reply:
I think `any` and `all` might be good labels too.

Reply:
so remove `or` `and` instead of `any` and `all`?

Reply:
or do you want all of them `or`, `and`, `any`, `all`

Reply:
okay cool. Ill change this up